js/utils.js

```diff
@@ -45,6 +45,71 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     return parseFloat(value);
   }
 
+  function getFilesInfo(options) {
+    var entry = options.entryData;
+    var detailViewFileOptions = _.filter(options.detailViewOptions, function(option) {
+      return option.editable && option.type === 'file';
+    });
+
+    var formFilesInfoInDetailViewOptions = detailViewFileOptions.map(function(detailViewFileOption) {
+      return new Promise(function(resolve) {
+        var label = '';
+        var labelEnabled = true;
+        var files = typeof entry.originalData[detailViewFileOption.column] === 'string' ?
+          splitByCommas(entry.originalData[detailViewFileOption.column]) :
+          entry.originalData[detailViewFileOption.column];
+
+        if (!files) {
+          return resolve();
+        }
+
+        switch (detailViewFileOption.fieldLabel) {
+          case 'column-name':
+            if (detailViewFileOption.column !== 'custom') {
+              label = detailViewFileOption.column;
+            }
+
+            break;
+          case 'custom-label':
+            label = new Handlebars.SafeString(Handlebars.compile(detailViewFileOption.customFieldLabel)(entry.originalData));
+
+            break;
+          case 'no-label':
+            labelEnabled = false;
+
+            break;
+          default:
+            break;
+        }
+
+        var filesDetailViewInfo = Promise.all(files.map(function(fileUrl) {
+          var fileId = Fliplet.Media.isRemoteUrl(fileUrl)[2];
```

Comment:
It's sematically weird to use `Fliplet.Media.isRemoteUrl()` to get the file ID. I understand it uses regex so this will probably work. @squallstar Do you approve?

Reply:
1. This will break if `isRemoteUrl` doesn't return an array
2. This will break if we change the capturing groups (or their order) in `isRemoteUrl`. Technically, this will be more solid if it doesn't rely on it and uses a simple regexp to grab the ID, e.g. `v1\/media\/files/([0-9]+)`

Reply:
@squallstar Thanks for your input.

@YaroslavOvdii Please use a regexp to directly grab the file ID instead of using `Fliplet.Media.isRemoteUrl()`